feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -80,7 +83,7 @@ <h2 aria-hidden="true" id="{{ getGroupId(idPrefix, group.id) }}">
             context: {
               overviewGroups: group.subGroups,
               level: level + 1,
-              idPrefix: idPrefix + group.id
+              idPrefix: idPrefix ? idPrefix + '--' + group.id : group.id
```

Comment:
Why was that needed (just because I am curious)?

Reply:
can we move this logic to configurator-storefront-utils, as well?
it duplicated inn the configurator-overview-menu component. Also the seperator
'--' is hard coded in several locations. Would be easier if all is encapsulated in one service class (configurator-storefront-utils)

Reply:
In order to better map a group to the corresponding menu item, for instance:
**group:** KD990WRES--KD990WRESP--KD990WRESPS--KD990WRESP1--TECH**-ovGroup**
**menu item:** KD990WRES--KD990WRESP--KD990WRESPS--KD990WRESP1--TECH**-ovMenuItem**

We have to only cut a suffix **-ovGroup** und replace by **-ovMenuItem**